code/_onclick/other_mobs.dm

```diff
@@ -127,22 +127,10 @@
 	delayNextAttack(10)
 	if(istype(wear_mask, /obj/item/clothing/mask/muzzle))
 		return
+
 	var/mob/living/carbon/ML = A
-	var/dam_zone = ran_zone(pick("chest", "l_hand", "r_hand", "l_leg", "r_leg"))
```

Comment:
That change doens't affect the game at all, I just removed duplicate code
And it only affects handcuffed monkeys